report/telegram.go

```diff
@@ -0,0 +1,67 @@
+package report
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/future-architect/vuls/config"
+	"github.com/future-architect/vuls/models"
+)
+
+type TelegramWriter struct{}
```

Comment:
__[Golint]__ exported type TelegramWriter should have comment or be unexported

<sub>posted by Sider</sub>